learn/advanced/updating.md

```diff
@@ -225,35 +176,192 @@ Once the `dumpCreation` task shows `"status": "succeeded"`, you're ready to move
 
 ::: note
 If you are using the Meilisearch official images on DigitalOcean, AWS, or GCP, you will find the `data.ms` folder at `/var/lib/meilisearch/data.ms`.
 :::
 
-## Step 6: Import the dump
+Move the binary of the current Meilisearch version and the database to the `tmp/` folder:
+
+:::: tabs
+
+::: tab Local installation
+
+```
+mv /path/to/your/meilisearch/directory/meilisearch /tmp
+mv /path/to/your/meilisearch/directory/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::: tab Cloud platforms
+
+```
+mv /usr/bin/meilisearch /tmp
+mv /var/lib/meilisearch/data.ms /tmp/
+```
+
+:::
+
+::::
+
+## Step 6: Install the desired version of Meilisearch
+
+Install the latest version of Meilisearch using:
+
+:::: tabs
+
+::: tab Local installation
+
+```bash
+# Install Meilisearch
+curl -L https://install.meilisearch.com | sh
+
+# Launch Meilisearch
+./meilisearch
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {meilisearch_version} with the version of your choice. Use the format: `vX.X.X`
+curl "https://github.com/meilisearch/meilisearch/releases/download/{meilisearch_version}/meilisearch-linux-amd64" --output meilisearch --location --show-error
+```
+
+:::
+
+::::
 
-Now that you’ve got your dump, install the [latest version of Meilisearch](/learn/getting_started/quick_start.md#setup-and-installation) and [import the dump](/learn/advanced/dumps.md#importing-a-dump) at launch using the CLI option:
+Give execute permission to the Meilisearch binary:
+
+```
+chmod +x meilisearch
+```
+
+For cloud platforms, move the new Meilisearch binary to the `/usr/bin` directory containing the executable files:
+
+```
+mv meilisearch /usr/bin/meilisearch
+```
+
+## Step 7: Launch Meilisearch and import the dump
+
+Execute the command below to import the dump at launch:
+
+:::: tabs
+
+::: tab Local installation
 
 ```bash
-# launch the latest version of Meilisearch with the master key and import the specified dump file
-./meilisearch --import-dump /dumps/your_dump_file.dump --master-key="MASTER_KEY"
+# replace {dump_uid.dump} with the name of your dump file
+./meilisearch --import-dump /dumps/{your_dump_file.dump} --master-key="MASTER_KEY"
+```
+
+:::
+
+::: tab Cloud platforms
+
+```sh
+# replace {dump_uid.dump} with the name of your dump file
+meilisearch --db-path /var/lib/meilisearch/data.ms --import-dump "/var/opt/meilisearch/dumps/{dump_uid.dump}"
 ```
 
+:::
+
+::::
+
 ::: warning
 If you are using Meilisearch v0.20 or below, migration should be done in two steps. First, import your dump into an instance running any version of Meilisearch from v0.21 to v0.24, inclusive. Second, export another dump from this instance and import it to a final instance running your targeted version.
+
+Once Meilisearch v1 is released, this two-step process won't be necessary as v1 will be compatible with dumps from all previous versions.
 :::
```

Comment:
Feels like this warning needs to come sooner, perhaps in the previous step: install the desired version of Meilisearch.